frontend/device/kindle/device.lua

```diff
@@ -638,7 +638,7 @@ local KindleScribe = Kindle:extend{
     hasLightSensor = yes,
     hasGSensor = yes,
     display_dpi = 300,
-    touch_dev = "/dev/input/by-path/platform-1001e000.i2c-event",
+    touch_dev = "/dev/input/touch",
```

Comment:
This is better on all devices? It looks a bit suspicious as a diff.

Reply:
I don't know about any other device, but this is the only solution for Kindle Scribe.

Reply:
P.S. NiLuJe and I discussed this in length today from https://github.com/koreader/koreader/issues/11199#issuecomment-1870512410 and on

Reply:
I'd missed that it wasn't in the generic definition through the mobile diff.